src/main/java/cn/nukkit/command/defaults/SetWorldSpawnCommand.java

```diff
@@ -35,7 +35,7 @@ public boolean execute(CommandSender sender, String commandLabel, String[] args)
         if (args.length == 0) {
             if (sender instanceof Player) {
                 level = ((Player) sender).getLevel();
-                pos = ((Player) sender).getPosition().round();
+                pos = ((Player) sender).getPosition().floor();
```

Comment:
This doesn't need to be changed.

Reply:
By using floor it will always use the block you are standing in.  If you use round it may return the neighboring block

Reply:
removed the `.floor()`  in the next commit call as it's not needed